docs/quickstart-rosa.md

```diff
@@ -191,7 +191,15 @@ ROSA can be installed using command line parameters or in interactive mode.  For
   Pod CIDR [10.128.0.0/14]:
   Host prefix [23]:
   Private cluster (y/N):
-  ```  
+  ```
+  
+  You must create operator-roles and oidc-provider as shown in the above create cluster command's output. For e.g.
```

Comment:
this shouldn't actually be necessary, I think we might need to update the create command to include `--mode auto` which should do these steps for us.